src/utils/AddMember/AddMember.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import Button from '../../components/Button/Button'
+import { FlexCol, FormBox } from './AddMember.styles'
+
+const AddMember = () => {
+    return (
+        <div>
+
+            <FormBox>
+                <h2>Add Member Form</h2>
+                <FlexCol>
```

Comment:
createAddMemberFormField = ({name, type}) ={
return (<FlexCol>
                    <div className='form-row'>
                        <label htmlFor="">{name}</label>
                        <input type={type} />
                    </div>
                             </FlexCol>
)
}

you can even create a state object with form fields name and type and loop it over such that,
addMemberForm.forEach((field)=>{
const {name,type} = field;
createAddMemberFormField(name, type)
})